includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -1,21 +1,271 @@
 <?php
 
 class DesignSystemApiController extends WikiaApiController {
+	private $data = [
+		"logo" => [
+			"header" => [
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "header" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)